.github/workflows/asset-size-check.yml

```diff
@@ -34,21 +34,21 @@ jobs:
       - name: Checkout master
         run: git checkout master
       - name: Install dependencies for master
-        run: yarn install
+        run: pnpm install
```

Comment:
update cache above

Reply:
all of the workflows likely have caches to update btw, just pointed out the first couple